python/pyspark/pandas/supported_api_gen.py

```diff
@@ -142,7 +142,7 @@ def _create_supported_by_module(
         [
             m
             for m in getmembers(pd_module, isfunction)
-            if not m[0].startswith("_") and m[0] in pd_module.__dict__
```

Comment:
`inspect.getmembers` returns all functions supported by the module, including those inherited from parents, but `__dict__` only returns functions directly implemented in the module, excluding inherited functions.

Therefore, the inherited and implemented API was treated as not supported in the existing script.